server/handlers/chat_handler.js

```diff
@@ -1,32 +1,14 @@
-import { ConversationChain } from 'langchain/chains';
-import { ChatOpenAI } from 'langchain/chat_models/openai';
-import { ChatPromptTemplate, HumanMessagePromptTemplate, MessagesPlaceholder, SystemMessagePromptTemplate } from 'langchain/prompts';
-import { ConversationSummaryMemory } from 'langchain/memory';
+import { MODEL_STORES, enabledModel } from '../config/model_store_constants.js';
 
 class ChatService {
   constructor () {
-    this.chat = new ChatOpenAI({ temperature: 0, verbose: true });
-    this.chatPrompt = ChatPromptTemplate.fromPromptMessages([
-      SystemMessagePromptTemplate.fromTemplate('The following is a friendly conversation between a human and an AI. The AI is talkative and provides lots of specific details from its context. If the AI does not know the answer to a question, it truthfully says it does not know.'),
-      new MessagesPlaceholder('history'),
-      HumanMessagePromptTemplate.fromTemplate('{input}'),
-    ]);
-
-    this.memory = new ConversationSummaryMemory({ llm: this.chat, returnMessages: true });
+    this.model = new MODEL_STORES[enabledModel]
```

Comment:
How do you think we should control temperature, verbose, and other props that get passed into the models?

Reply:
Let's revisit when there are more models to use. Maybe they will mostly follow the same structure.